climada/hazard/test/test_tc_surge_bathtub.py

```diff
@@ -69,9 +69,9 @@ def __enter__(self):
             'nodata': -32767.0,
         }
 
-        # In Windows, unlike Unix, the temporary file cannot be opened before it is closed
+        # In Windows, unlike Unix, the temporary file cannot be opened before it is closed.
         # Therefore it is closed right after creation and only the path/name is kept.
-        tmpfile = tempfile.NamedTemporaryFile()
+        tmpfile = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
By default, the file is deleted when closed which is not what we want because we close it right away.